src/demos/Sliders.vue

```diff
@@ -1,175 +1,203 @@
 <template>
   <div>
     <h1>Sliders</h1>
-
+    <!-- General / Intro -->
     <div class="section">
-      <h2>Default Slider</h2>
-      <p>
-        Our slider component is extended from
-        <a
-          target="_blank"
-          href="https://github.com/NightCatSama/vue-slider-component"
-          >Vue Slider Component</a
-        >
-      </p>
-      <pre><code>import { Slider } from 'streamlabs-beaker';
+      <div class="row">
+        <p>
+          Our slider component is extended from
+          <a
+            target="_blank"
+            href="https://github.com/NightCatSama/vue-slider-component"
+            >Vue Slider Component</a
+          >
+          and makes use of
+          <a
+            target="_blank"
+            href="https://github.com/que-etc/resize-observer-polyfill"
+            >Resize Observer Polyfill</a
+          >.
+        </p>
+        <p>
+          This slider works best when using
```

Comment:
This doesn't really make much technical sense.  Please familiarize yourself with this section of the Vue docs: https://vuejs.org/v2/guide/forms.html

Reply:
thanks, I'll check it out, It didnt make sense to me either, I did not make the component that we extended, I just know that v-model worked right off the bat, while value and input needed considerable work to work, Its why i wanted a complete rewrite of slider, but Noone has time for that right now.

Reply:
Hmm that's pretty strange.  Would be curious to see why that is.  The reason why `:value` and `@input` is required if you're using a flux-like store such as Vuex.  Using `v-model` breaks your unidirectional data flow, so it's pretty important.

Reply:
ill get you the posts from the author in a few mins

Reply:
Realized that I posted the wrong section of the docs.  Since this is a custom component, this is the correct section.  https://vuejs.org/v2/guide/components.html#Using-v-model-on-Components